src/pool.c

```diff
@@ -30,7 +30,7 @@ struct mrb_pool_page {
   size_t offset;
   size_t len;
   void *last;
-  char page[];
+  char page[1];
```

Comment:
This makes allocation size bigger by 1 byte.

Reply:
Shouldn't `page_alloc()` use `offsetof(struct mrb_pool_page, page)` instead of `sizeof(struct mrb_pool_page)` together?

Reply:
This simply have to disable C4100.

```
#pragma warning(disable : 4200) 
```
see https://msdn.microsoft.com/en-us/library/79wf64bc.aspx

>This non-standard extension is often used to interface code with external data structures that have a variable length. If this scenario applies to your code, you can disable the warning:

Reply:
Fixed, used `#pragma warning` with MSVC.